Makefile

```diff
@@ -135,7 +135,7 @@ BASE_IMAGES_TAG := $(shell awk '/base_images_tag/ \
 	{split($$NF, a, "\""); print a[2]}' base-images-tag.json)
 
 test-images: docker-images presto-server-rpm.rpm
-	python tests/product/image_builder.py $(IMAGE_NAMES)
+	PYTHONPATH=${PYTHONPATH}:$(shell pwd) python tests/product/image_builder.py $(IMAGE_NAMES)
```

Comment:
I think it works for you because you have presto admin installed in your virtualenv. Can you please do `pip uninstall prestoadmin` and try again?

I think having presto-admin installed in virtualenv could be misleading as code changes wont be seen until you update presto in virtualenv (you need to do `pip install` or something similar to make build script be affected by the changes).